crates/rpc/src/v06/method/simulate_transactions.rs

```diff
@@ -684,9 +684,9 @@ pub(crate) mod tests {
             SimulatedTransaction {
                 fee_estimation:
                     FeeEstimate {
-                        gas_consumed: 3097.into(),
+                        gas_consumed: 2222.into(),
```

Comment:
There was some rust crate that made these sort of test changes easier to manage. I forgot what its called though. I think it generates fixture files as a snapshot and then on test failure you can decide whether to update the snapshot or not.